### PR TITLE
Feature/bal 20 gp update password

### PR DESCRIPTION
**Pull request by gperissetcelteeka**
# Password update

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
#20
#90 Title Change
#82 Wrong animation

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Balance/Fealing/FeelingView.swift [hunk 1]
```diff
@@ -31,14 +31,14 @@ struct FeelingView: View {
...
-                viewName: "Guess the Emoticon Feature",
+                viewName: "Guess the Emotion Feature",
                 isDirectChildToContainer: true,
                 content: {
                     GuessView()
                 }
             )
         ) {
-            DistractionCellView(image: "guessIcon", text: "Guess Emoticon", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
+            DistractionCellView(image: "guessIcon", text: "Guess the emoticon", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
```
Comment (sebastianricaldoni) [inline Balance/Fealing/FeelingView.swift:41]:
```suggestion
            DistractionCellView(image: "guessIcon", text: "Guess the emotion", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
```

File: Balance/Fealing/MoodEditorView.swift [hunk 2]
```diff
@@ -78,7 +78,7 @@ struct MoodEditorView: View {
                 }
             }
         }) {
-            Text("Save")
+            Text("Save in diary")
```
Comment (sebastianricaldoni) [inline Balance/Fealing/MoodEditorView.swift:81]:
```suggestion
            Text("Save in Diary")
```

File: Balance/Profile/PasswordUpdateView.swift [hunk 3]
```diff
@@ -19,41 +22,90 @@ struct PasswordUpdateView: View {
...
+                if $password.wrappedValue.isEmpty ||
+                    $confirmPassword.wrappedValue.isEmpty {
+                    self.alertMessage = "All the information is required"
+                    showingAlert = true
+                    return
+                }
+                
+                if $password.wrappedValue !=
+                    $confirmPassword.wrappedValue {
+                    self.alertMessage = "The confirmation password mismatch"
```
Comment (sebastianricaldoni) [inline Balance/Profile/PasswordUpdateView.swift:97]: I would use either `The confirmation password did not match` or `Confirmation password mismatch` @gperissetcelteeka

**Review by sebastianricaldoni — CHANGES_REQUESTED** [review]
Just a typo ...`emoticon` vs `emotion`

**Review by sebastianricaldoni — APPROVED** [review]
LGTM

File: Balance/Profile/PasswordUpdateView.swift (hunk 3 above)
Comment (gperissetcelteeka) [reply Balance/Profile/PasswordUpdateView.swift:97]: ok
